opentelemetry-user-events-logs/src/logs/exporter.rs

```diff
@@ -219,34 +219,29 @@ impl UserEventsExporter {
                 eb.add_str("time", time, FieldFormat::Default, 0);
 
                 if let Some(trace_context) = log_record.trace_context() {
-                    cs_a_count += 1; // for ext_dt
-                    // TODO: Flattened structure might be faster
-                    eb.add_struct("ext_dt", 2, 0);
-                    eb.add_str("traceId", trace_context.trace_id.to_string(), FieldFormat::Default, 0);
-                    eb.add_str("spanId", trace_context.span_id.to_string(), FieldFormat::Default, 0);
+                    cs_a_count += 2; // for ext_dt_traceId and ext_dt_spanId
+                    eb.add_str(
+                        "ext_dt_traceId",
```

Comment:
https://github.com/open-telemetry/opentelemetry-rust-contrib/blob/main/opentelemetry-user-events-logs/src/lib.rs#L337 
can you modify these tests too? they are not run in CI, so must be run locally to see which ones needs fixes with this change.

Reply:
cloud_rol also require test changes.